vkrunner/vr-box.c

```diff
@@ -223,11 +228,16 @@ compare_unsigned(enum vr_box_comparison comparison,
 static bool
 compare_double(enum vr_box_comparison comparison,
                double a,
-               double b)
+               double b,
+               double tolerance,
+               bool percent)
 {
         switch (comparison) {
         case VR_BOX_COMPARISON_EQUAL:
                 return a == b;
+        case VR_BOX_COMPARISON_FUZZY_EQUAL:
+                if (percent) tolerance = tolerance / 100.0 * b;
```

Comment:
I think we might need to wrap this in `fabs` in case `b` is negative.

Please split this into two lines.

Reply:
I was just asking not to put both the condition and the then-clause of the if-statement on the same line. Ie, it would be:

```C
if (percent)
        tolerance = tolerance / 100.0 * b;
```

But it’s irrelevant now because `vr_tolerance_equal` doesn’t have this line anymore.